avcheck.py

```diff
@@ -0,0 +1,606 @@
+                        'webroot', 'quarantine', 'virus', 'threat', 'infected'
+                    ]
+                    
+                    is_av_directory = any(av_dir in file_name_lower for av_dir in av_directories)
+                    
+                    # Exclude common false positives
+                    false_positives = ['windowsupdate.log', 'iconcache.db', 'thumbs.db', 
+                                     'prefetch', 'recent', 'cookies']
+                    is_false_positive = any(fp in file_name_lower for fp in false_positives)
+                    
+                    if (is_detection_file or is_av_directory) and not is_false_positive:
+                        # Determine file type with more granularity
+                        if any(p in file_name_lower for p in ['quarantine', 'infected', 'malware']):
+                            file_type = "Quarantine"
+                        elif any(p in file_name_lower for p in ['detection', 'threat']):
+                            file_type = "Detection Log"
+                        elif 'scan' in file_name_lower:
+                            file_type = "Scan Log"
+                        elif any(p in file_name_lower for p in ['mpenginedb', 'signature', 'definition']):
+                            file_type = "Signature DB"
+                        else:
+                            file_type = "AV Artifact"
+                        
+                        # Extract threat information from filename if present
+                        threat_indicators = []
+                        threat_keywords = ['trojan', 'virus', 'malware', 'ransomware', 
+                                         'backdoor', 'rootkit', 'worm', 'spyware']
+                        
+                        for keyword in threat_keywords:
+                            if keyword in file_name_lower:
+                                threat_indicators.append(keyword.capitalize())
+                        
+                        threat_info = ', '.join(threat_indicators) if threat_indicators else 'N/A'
+                        
+                        av_files.append({
+                            'path': file_name,
+                            'type': file_type,
+                            'threats': threat_info,
+                            'offset': file_obj.vol.offset
+                        })
+                        
+                except Exception as e:
+                    vollog.debug(f"Error processing file: {e}")
+                    continue
+                    
+        except Exception as e:
+            vollog.debug(f"Error in filescan: {e}")
+        
+        return av_files
+    
+    def _identify_av_vendor(self, proc_name: str, proc_path: str) -> str:
+        """Identify the AV vendor from process name/path."""
+        proc_lower = (proc_name + ' ' + proc_path).lower()
+        
+        vendor_checks = [

```

Comment:
Again, another data dump, please put this into a JSON file and load it at the beginning.  It also isn't clear why this is a `tuple[list[str], str]`, rather than just `dict[str, list[str]]`?